### ISSUE TITLE
VIS 1.5.4 Uncaught ReferenceError

### ISSUE THREAD

**Issue by ghost** [ioBroker/ioBroker.vis#816]
There is an error in console after action in vis runtime (e.g. push button).

![image](https://github.com/ioBroker/ioBroker.vis/assets/126241718/1487ee4a-a8af-4806-a581-65cc4657a56c)

![image](https://github.com/ioBroker/ioBroker.vis/assets/126241718/31cbb0b2-849f-44f4-97d4-5ba73db2582f)

Ich arbeite mit VIS-1. Seit dem update auf Version 1.5.4 gibt es folgenden Fehler:

Ich öffne/starte ein Projekt im runtime; egal welches, auch ein "neues". Ich öffne mit F12 die Konsole -> keine Fehler. Ich führe irgendeine Aktion in der VIS aus (Button drücken oder ähnliches) und es gibt folgenden Fehler:

**Comment by mcm1957** [comment]
If I followed discussion at forum theres only an error in browser console.

Usage of vis and vis editor is not effected and works as expected.

Please confurm or correct.

I ask to check priority of issue.

**Comment by ghost** [comment]
Correct. Usage of vis and vis editor is not effected. Please set low priority of issue.
